shared/css/homepage_below_hero_pd_banner.scss

```diff
@@ -0,0 +1,178 @@
+$light_background_colour: #a69bc1;
+$blue_background_colour: #59b9dc;
+$gold_background_colour: #ffdfa1;
+$dark_blue_background_colour: #0d4172;
+
+@keyframes slideInFromLeft {
+  0% {
+    transform: translateX(-50px);
+  }
+  100% {
+    transform: translateX(0);
+  }
+}
+
+.animateSlideInFromLeft {
+  animation: 2s ease-out 0s 1 slideInFromLeft;
+}
+
+.homepage-professional-learning-banner {
+  margin-top: 30px;
+  margin-bottom: 30px;
+
+  a {
+    font-family: "Gotham 4r", sans-serif;
+    text-decoration: none;
+  }
+
+  button {
+    font-size: 18px;
+    min-width: 140px;
+    background-color: transparent;
+    border-color: white;
+    border-width: 2px;
+  }
+
+  .header {
+    animation: fadein 2s;
+    font-size: 18px;
+  }
+
+  .text {
+    animation: fadein 2s;
+    font-size: 14px;
+  }
+
+  .homepage-pd-banner-desktop {
+    background-color: $gold_background_colour;
+    height: 97px;
+    margin: 0 auto;
+    position: relative;
+    width: 100%;
+
+    .banner {
+      text-align: center;
+      margin: 0 auto;
+      float: right;
+
+      img {
+        width: 600px;
+      }
+    }
+
+    .left {
+      float: left;
+      height: 87px;
+      position: absolute;
+      width: 50%;
+
+      .header {
+        color: black;
+        padding: 15px 10px 10px 30px;
+        text-align: left;
+        position: relative;
+      }
+
+      .text {
+        color: black;
+        text-align: left;
+        line-height: 1;
+        position: relative;
+        left: 30px;
+        width: calc(100% - 150px);
+      }
+    }
+
+    .right {
```

Comment:
It's interesting that this class is called `right` and the first rule is `float: left`, is this rule needed? The `right` class is definitely worth auditing to see if you can simplify the css! Let me know if you want to pair on this

Reply:
Good catch- this line (along with height) wasn't necessary anymore, so they've been removed. Totally open to hear more thoughts on simplifying though!